modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java

```diff
@@ -913,6 +924,18 @@ _liferayPortletRequest, getSearchSearchContainerURL(), null,
 		return searchContainer;
 	}
 
+	private boolean _isAncestor(FileEntry fileEntry, long parentFolderId) {
```

Comment:
I think the function name should make more sense. May `_isAncestorFolder(long folderId, FilleEntry fileEntry)`? What do you think?

Reply:
I updated it, thank you!